environment.yml

```diff
@@ -3,6 +3,7 @@ channels:
   - conda-forge
   - nodefaults
 dependencies:
+  - python<3.10
```

Comment:
What is the issue with 3.10? Can you add a comment about that here?

Reply:
Seems that the incompatibility between python 3.10, sphinx, and readthedocs has been fixed. I re-ran PR #170 and it was successful. No need for this PR anymore. glum is also passing with python 3.10 so I will be changing it there as well.